hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2855,4 +2854,11 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.client.key.latest.location.version</name>
```

Comment:
NIT: should we rename it to ozone.client.key.latest.version.location?

Reply:
@xiaoyuyao Thanks for the review. Updated the config key.